Objects/codeobject.c

```diff
@@ -11,9 +11,23 @@
 #include "pycore_tuple.h"         // _PyTuple_ITEMS()
 #include "clinic/codeobject.c.h"
 
+static PyObject* code_repr(PyCodeObject *co);
+
+static const char *
+code_event_name(PyCodeEvent event) {
```

Comment:
FWIW, I'm not sure this stringification is worth the effort (I would just give the enums an explicit value, so you can manually map back, and log the integer), but this may make more sense as an inline function in codeobject.h so the FOREACH_CODE_EVENT macro doesn't have to leave the header.

Reply:
Given that `FOREACH_FUNC_EVENT` already has to escape the header, do you think it's worth having `FOREACH_DICT_EVENT` and `FOREACH_CODE_EVENT` not escape, or better to handle all of them the same way? I'd lean towards consistency in all three cases.

Reply:
Yeah, there's enough subtle differences between the three systems already, let's keep them as similar as we can.